source/MRMeshC/MRMeshBuilder.cpp

```diff
@@ -0,0 +1,44 @@
+#include "MRMeshBuilder.h"
+#include "MRMesh/MRMesh.h"
+#include "MRMesh/MRMeshBuilder.h"
+
+#include "detail/TypeCast.h"
+#include "detail/Vector.h"
+
+using namespace MR;
+
+REGISTER_AUTO_CAST( Mesh )
+REGISTER_VECTOR( VertMap )
```

Comment:
mb this vector should be registered separately (it seems to be useful in other places too)